src/types/index.ts

```diff
@@ -7,6 +7,8 @@ export interface ConsentState {
 }
 export interface PubData {
 	browserId?: string;
+	pageViewId?: string;
+	cmpInitTimeUtc?: number;
```

Comment:
is there a more accurate type you could use here?

Reply:
@sndrs What were you imaginging? Such as a DateTime? It's essentially a long

Reply:
yeah true, i guess it's just an infinite number of milliseconds?